Lesson-3/src/Lesson3.java

```diff
@@ -0,0 +1,97 @@
+public class Lesson3 {
+
+    public static void printThreeWords (){
+        System.out.println("Orange\nBanana\nApple");
+    }
+
+    public static void checkSumSign (int a, int b){
+
+        if ((a+b) >= 0){
+            System.out.println("Сумма положительная");
+        } else {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+
+    public static void printColor (int value){
+
+        if (value <=0){
+            System.out.println("Красный");
+        } else if (value > 0 & value <=100 ) {
+            System.out.println("Желтый");
+
+        } else if (value > 100) {
+            System.out.println("Зеленый");
+        }
+    }
+
+    public static void compareNumbers (int a, int b){
+        if (a>=b){
+            System.out.println("a>=b");
+        } else {
+            System.out.println("a<b");
+        }
+    }
+
+    public static boolean sum (int a, int b){
+        int sum = a+b;
+        if (sum>=10 & sum <=20){
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Всегда есть более коротки способы. Опять же, рекомендую посмотреть разбор домашки.
```suggestion
        return sum>=10 & sum <=20;
```